test-utils/build.gradle

```diff
@@ -1,8 +1,26 @@
-apply plugin: 'java'
+apply plugin: 'com.android.library'
```

Comment:
Is changing this module necessary? I see where we use `BuildConfig.VERSION_NAME` in the `networking` module but not clear where we use it in `test-utils`.

Reply:
`test-utils` depends on `networking`, so it can't be a vanilla java module 😞 this will be a change to make down in the future